pkgs/by-name/ya/yazi-unwrapped/package.nix

```diff
@@ -25,6 +25,8 @@ rustPlatform.buildRustPackage rec {
 
   env.YAZI_GEN_COMPLETIONS = true;
 
+  cargoBuildFlags = ["-p" "yazi-fm" "-p" "yazi-cli"];
```

Comment:
I have added it to `default-members`, so we can remove this next version.

https://github.com/sxyazi/yazi/pull/944/files

Reply:
So do you think we should just wait for the next release or do we add a message like this:

```nix
  # remove in the next release
  cargoBuildFlags = ["-p" "yazi-fm" "-p" "yazi-cli"];
```